ci/release-image/entrypoint.sh

```diff
@@ -5,15 +5,15 @@ set -eu
 # Otherwise the current container UID may not exist in the passwd database.
 eval "$(fixuid -q)"
 
-if [ "${DOCKER_USER-}" ] && [ "$DOCKER_USER" != "$USER" ]; then
+USER="$DOCKER_USER"
```

Comment:
Should we put this behind an `if [ "${DOCKER_USER-}" ]` in case `DOCKER_USER` is not defined?

Reply:
We can certainly do so, although entrypoint.sh should assume this env variable is defined when running the container. We should check it just in case it's not.